### ISSUE TITLE
Pyre ignores incompatible type in assignment for class attributes

### ISSUE THREAD

**Issue by linnik** [facebook/pyre-check#94]
```python
class Sample:

    size: str = 0  # this incompatible type in assignment is ignored by Pyre

    def __init__(self, size: int) -> None:
        self.size = size  # this one is catched as expected

```
Only one type error is found by Pyre:
```
(env) ~/w/test ❯❯❯ pyre --version
pyre version 0.0.10
(env) ~/w/test ❯❯❯ pyre check                                                                                                                                                                                                            ✘ 1
 ƛ Found 1 type error!
sample.py:6:8 Incompatible attribute type [8]: Attribute `size` declared in class `sample.Sample` has type `str` but is used as type `int`.
```
For the sake of testing, here is how mypy handles it:
```
(env) ~/w/test ❯❯❯ mypy sample.py                                                                                                                                                                                                        ✘ 1
sample.py:3: error: Incompatible types in assignment (expression has type "int", variable has type "str")
sample.py:6: error: Incompatible types in assignment (expression has type "int", variable has type "str")
```

**Comment by dark** [comment]
Hi @linnik, thanks for the report.

Incompatible types at class scope are still ignored right now. We are planning to include support for it at some point, but it's not scheduled yet. As always, patches are welcome :)

**Comment by MaggieMoss** [comment]
This is supported in the most recent Pyre releases. Thanks again for flagging :)
